src/utils/Time.cpp

```diff
@@ -0,0 +1,20 @@
+/**
+ * \file Time.cpp
+ *
+ * \copyright Copyright (C) 2019 Kano Computing Ltd.
+ *            License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+ *
+ * \brief     TODO
+ */
+
+
+#include <chrono>
+
+using std::chrono::duration_cast;
+using std::chrono::milliseconds;
+using std::chrono::system_clock::now;
+
+
+double getTimestamp() {
+    return duration_cast<milliseconds>(now().time_since_epoch()).count();
```

Comment:
Do we want to return a time type instead of `double`?

Reply:
We could but I wanted to KISS this function so it's nice and easy to call. A romantic at heart, you see.